src/ol/layer/Heatmap.js

```diff
@@ -282,7 +284,7 @@ class Heatmap extends BaseVector {
       .setStrokeColorExpression(
         `vec4(smoothstep(0., 1., (1. - length(currentRadiusPx * 2. / v_width)) * getBlurSlope()) * ${weightExpression})`,
       )
-      .setStrokeWidthExpression(`(${radiusCompiled} + ${blurCompiled})`)
+      .setStrokeWidthExpression(`(${radiusCompiled} + ${blurCompiled}) * 2.`)
```

Comment:
I added a 2 factor for the line width formula otherwise the use of the `radius` and `blur` parameters wasn't consistent between point and lines